source/client/client.cc

```diff
@@ -89,7 +89,7 @@ bool Main::run() {
     ENVOY_LOG(error, "An error occured while formatting proto");
     result = false;
   }
```

Comment:
Oh I think we need:
```c++
if (!formatted_proto.ok()) {
  // .... <snip> looks ok to me as-is.
} else {
  // only when formatted_proto.ok() == true
  std::cout << *formatted_proto;
}
```
I'm sorry if my earlier comment here was off or caused confusion.